dev/io.openliberty.checkpoint/src/io/openliberty/checkpoint/internal/CheckpointImpl.java

```diff
@@ -27,28 +28,36 @@
 import org.osgi.service.component.annotations.ReferenceCardinality;
 import org.osgi.service.component.annotations.ReferencePolicy;
 import org.osgi.service.component.annotations.ReferencePolicyOption;
+import org.osgi.framework.Constants;
 
 import com.ibm.websphere.ras.Tr;
 import com.ibm.websphere.ras.TraceComponent;
 import com.ibm.wsspi.kernel.service.location.WsLocationAdmin;
 import com.ibm.wsspi.kernel.service.location.WsLocationConstants;
+import com.ibm.ws.runtime.update.RuntimeUpdateListener;
+import com.ibm.ws.runtime.update.RuntimeUpdateManager;
+import com.ibm.ws.runtime.update.RuntimeUpdateNotification;
+import com.ibm.ws.threading.listeners.CompletionListener;
+import com.ibm.ws.kernel.boot.internal.BootstrapConstants;
+import com.ibm.ws.kernel.feature.ServerReadyStatus;
 
 import io.openliberty.checkpoint.internal.criu.ExecuteCRIU;
-import io.openliberty.checkpoint.internal.Checkpoint;
 import io.openliberty.checkpoint.spi.CheckpointHook;
 import io.openliberty.checkpoint.spi.CheckpointHookFactory;
 import io.openliberty.checkpoint.spi.CheckpointHookFactory.Phase;
-import io.openliberty.checkpoint.spi.SnapshotFailed;
-import io.openliberty.checkpoint.spi.SnapshotFailed.Type;
+import io.openliberty.checkpoint.internal.CheckpointFailed;
+import io.openliberty.checkpoint.internal.CheckpointFailed.Type;
 
 @Component(
            reference = @Reference(name = "hookFactories", service = CheckpointHookFactory.class, cardinality = ReferenceCardinality.MULTIPLE, policy = ReferencePolicy.DYNAMIC,
                                   policyOption = ReferencePolicyOption.GREEDY),
-           property = { "osgi.command.function=snapshot", "osgi.command.scope=criu" })
-public class CheckpointImpl implements Checkpoint {
+           property = { "osgi.command.function=snapshot", "osgi.command.scope=criu", Constants.SERVICE_RANKING + ":Integer=-10000" })
```

Comment:
We should get rid of the `osgi.command` service properties at this point.  There is no need to keep on maintaining this test gogo command for doing a checkpoint.